### DISCUSSION TITLE
What exactly is "migration.create"?

### DISCUSSION THREAD

**Discussion by ScriptHUBofficial** [community/community#132635] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

Yesterday, I reset my password because I suspected that there might be unauthorized access to my account. After that, I noticed a "migration.create" entry in the security log. I did not initiate this action. What does this action mean? Is this an automatic action performed by GitHub, or is someone else trying to steal my account and transfer my repositories to their own account? Please provide information about this issue.

**Comment by davevad93** [comment]
Hi @ScriptHUBofficial ,

The GitHub docs states the following about [migration.create](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/security-log-events#migration):

- ***A migration file was created for transferring data from a source location (such as a GitHub.com organization or a GitHub Enterprise Server instance) to a target GitHub Enterprise Server instance.***

Usually when an event has been triggered by GitHub, something like this is displayed in the security log: 

![ooo](https://github.com/user-attachments/assets/b12283cf-ee9f-45bc-be2a-d5379c626d3c)

On the other hand, if your account name appears next to the event and you haven't triggered it, it's very likely that someone who has accessed your account did it.

**Reply by ScriptHUBofficial** [reply]
> Hi @ScriptHUBofficial ,
> 
> The GitHub docs states the following about [migration.create](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/security-log-events#migration):
> 
> * _**A migration file was created for transferring data from a source location (such as a GitHub.com organization or a GitHub Enterprise Server instance) to a target GitHub Enterprise Server instance.**_
> 
> Usually when an event has been triggered by GitHub, something like this is displayed in the security log:
> 
> ![ooo](https://private-user-images.githubusercontent.com/94308234/349929318-b12283cf-ee9f-45bc-be2a-d5379c626d3c.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzMDU0MzgsIm5iZiI6MTcyMTMwNTEzOCwicGF0aCI6Ii85NDMwODIzNC8zNDk5MjkzMTgtYjEyMjgzY2YtZWU5Zi00NWJjLWJlMmEtZDUzNzljNjI2ZDNjLkpQRz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE4VDEyMTg1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJhMzE1Mzk2NzE4NDQ4YzQ0ZjI2MDg5YWUyZDc3MTc5ZmI3ZGQwYzc3MTljMzM4ZDE5M2VlNTNiYmViZjhkMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0H_S0a-hnVd_duG9kSXhQbAexAPQ6cjxgKUxyfkEiQM)
> 
> On the other hand, if your account name appears next to the event and you haven't triggered it, it's very likely that someone who has accessed your account did it.

I think I remember now. Yesterday, I observed that the last update date of a repository on my account was 2 hours ago, even though I didn't update it and there was no change in the repository. That's why I requested a security log from GitHub. Maybe this request triggered such an event. Could it be that it was considered a migration when I requested the security log?

**Reply by davevad93** [reply]
@ScriptHUBofficial , if you exported the log from the [Security Log](https://github.com/settings/security-log), the event should be `user.audit_log_export` not `migration.create`.
If I'm not mistaken you can also request a security log directly from [GitHub Support](https://support.github.com/contact?flow=log_request&tags=rr-log-req), but I've never done it so I don't know what event it triggers in the log.

**Reply by ScriptHUBofficial** [reply]
> @ScriptHUBofficial , if you exported the log from the [Security Log](https://github.com/settings/security-log), the event should be `user.audit_log_export` not `migration.create`. If I'm not mistaken you can also request a security log directly from [GitHub Support](https://support.github.com/contact?flow=log_request&tags=rr-log-req), but I've never done it so I don't know what event it triggers in the log.

Unfortunately, support responds very slowly. A few years ago, my account was closed for an incorrect reason, and it took 2 months to get a response. I don't know what to do in such an important and worrying situation. Currently, all security measures on my account are active, and I have updated the necessary information. Unfortunately, I have no choice but to wait for now.

**Reply by davevad93** [reply]
Yes @ScriptHUBofficial , make sure you've set a very strong password and use a 2FA auth method to improve your account security and check from time to time your [active sessions](https://github.com/settings/sessions) if there's a session you don't recognize, revoke it. Also check in your [security log](https://github.com/settings/security-log) the IP of the events, if you see an IP or location that you don't recognize that's proof enough of an unauthorized access.
